### DISCUSSION TITLE
The plugin has stopped working for me. Is that the case for anyone else?

### DISCUSSION THREAD

**Discussion by tamdabam** [hadynz/obsidian-kindle-plugin#68] [General]
The plugin has stopped working for me. Is that the case for anyone else? 

I never encounter an error, just no books sync. 

Absolutely love the plugin, thanks for all your work guys!

**Comment by tamdabam** [comment]
Never mind.... I didn't realise the books stop syncing once you edit the files. 

Do you guys tend to copy the notes over to a different file and finalise there?

**Comment by hadynz** [comment]
You are running into a feature request about better support for "resyncing" of notes. You can follow [this long thread][1] that discusses some of the possible options to implement this feature.

[1]: https://github.com/hadynz/obsidian-kindle-plugin/discussions/27
